examples/goldilocks_models/initial_guess.cc

```diff
@@ -0,0 +1,195 @@
+//
+// Created by jianshu on 3/25/20.
+//
+#include "examples/goldilocks_models/initial_guess.h"
+
+namespace dairlib::goldilocks_models {
+//edited by Jianshu to try a new way of setting initial guess
+
+MatrixXd get_theta_scale(const string directory, int iter){
+//    VectorXd initial_theta = readCSV(directory + to_string(0) + string("_theta_s.csv"));
+    VectorXd current_theta = readCSV(directory + to_string(iter) + string("_theta_s.csv"));
```

Comment:
There seems to be a bug here. Theta should be `[theta_s; _theta_sDDot]^T`

Reply:
Oh, It seems like the weights will be changed due to my misunderstanding of the theta. I need to change all the thetas in this file.